### PR TITLE
feat/shadows-on-landing

### LINKED ISSUE #3835: Adjust shadows on landing page
I guess we should have shadows on the bottom right? As with other components, we already have it them that way 

image

like in chain selector for example 
image

### PR DESCRIPTION
**Pull request by KngZhi**
- feat: box-shadow for navbar

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3835
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

image

### CODE REVIEW

**Comment by roiLeo** [comment]
Oh no! I think @yangwao was asking to change box-shadow in this component
Screenshot 2022-08-26 at 10-33-41 KodaDot - Kusama NFT Market Explorer

instead to have top-right to bottom-left make it top-left to bottom-right

**Comment by KngZhi** [comment]
Okay, get it.

**Review by roiLeo — COMMENTED** [review]
missing box-shadow changes in light-mode?

**Comment by KngZhi** [comment]
let me move those into `_light` and `_dark`, so it would not be forgotten.

**Comment by roiLeo** [comment]
wdyt if we change it with `$dropdown-content-shadow`?

Screenshot 2022-08-26 at 11-22-19 KodaDot - Kusama NFT Market Explorer
Screenshot 2022-08-26 at 11-22-03 KodaDot - Kusama NFT Market Explorer

**Comment by yangwao** [comment]
> wdyt if we change it with `$dropdown-content-shadow`?

looks great!

File: styles/themes/_dark.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@ html.dark-mode {
 	.card-header-title,
 	.input {
 		color: $text;
+
```
Comment (roiLeo) [inline styles/themes/_dark.scss:16]:
```suggestion
```

File: styles/themes/_dark.scss [hunk 2]
```diff
@@ -21,4 +22,5 @@ html.dark-mode {
 			color: $black;
 		}
 	}
+
```
Comment (roiLeo) [inline styles/themes/_dark.scss:25]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by yangwao** [comment]
yas 
pay 50 

image

**Review by yangwao — APPROVED** [review]
<3

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 45.18 USD/KSM ~ 1.107 $KSM
🧗 [DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc](https://kusama.subscan.io/account/DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)
🔗 [0x1290d4a4095fcca9fb6764f0843880a75fbfef5d2b8120de810bd21a97ce32c4](https://kusama.subscan.io/extrinsic/0x1290d4a4095fcca9fb6764f0843880a75fbfef5d2b8120de810bd21a97ce32c4)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**